test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -144,38 +145,57 @@ public async Task VerifySignaturesAsync_ExpiredCertificateAndTimestamp_SuccessAs
             };
             var bcCertificate = ca.IssueCertificate(issueOptions);
 
-            using (var certificate = new X509Certificate2(bcCertificate.GetEncoded()))
             using (var directory = TestDirectory.Create())
             {
-                certificate.PrivateKey = DotNetUtilities.ToRSA(keyPair.Private as RsaPrivateCrtKeyParameters);
-                var notAfter = certificate.NotAfter.ToUniversalTime();
+#if IS_DESKTOP
+                var privateKey = DotNetUtilities.ToRSA(keyPair.Private as RsaPrivateCrtKeyParameters);
```

Comment:
Use explicit type instead of `var`.

Please apply this feedback appropriately throughout this PR.

Reply:
Thanks! Fixed.

Reply:
Thanks! Fixed.